site/tutorials/amqp-concepts.md

```diff
@@ -160,6 +160,9 @@ other words, the default exchange makes it seem like it
 is possible to deliver messages directly to queues, even
 though that is not technically what is happening.
 
+The default exchange, in RabbitMQ, does not allow bind/unbind operations.
+Binding operations to the default exchange will result in an error.
```

Comment:
```suggestion
Binding operations to the default exchange will result in an error.

In RabbitMQ, the default exchange is a specialised direct exchange which uses the routing key as the queue name.
```

Reply:
That's explained in the first paragraph of this section (it doesn't show in the git diff):

> The default exchange is a direct exchange with no name
(empty string) pre-declared by the broker. It has one special
property that makes it very useful for simple applications:
every queue that is created is automatically bound to it with
a routing key which is the same as the queue name.

Do you think we should re-iterate the same here?

Reply:
oh, no need then